templates/navigation/navbar.html

```diff
@@ -0,0 +1,53 @@
+<div class="navbar js-nav-expandable">
```

Comment:
Please remove this demo file. This is not needed in the pull request. Having the example markup in the pull request template is good enough.